src/App.spec.tsx

```diff
@@ -23,17 +25,6 @@ jest.mock('./pages/TermsOfService', () => () => 'TermsOfService')
 jest.mock('./pages/EnterpriseLandingPage', () => () => 'EnterpriseLandingPage')
 jest.mock('./pages/SyncProviderPage', () => () => 'SyncProviderPage')
 
-jest.mock('./shared/GlobalBanners', () => () => '')
```

Comment:
none of these mocks seemed to affect the suite

Reply:
just a heads up there's a failing test

Reply:
@RulaKhaled Haha yeah, called it out on slack that the mocks are being funky if you may have a workaround in your back pocket :O